Exercise_9/src/MyFramework/index.js

```diff
@@ -0,0 +1,16 @@
+/* global window:true */
+import App from './app';
+
+function MyFramework() {
```

Comment:
change function to class and use Symbols instead of constants to emulate private fields